prow/pod-utilities.md

```diff
@@ -82,6 +82,7 @@ to clone the repo to different go import path than the default of `/home/prow/go
 - Jobs that require additional repos to be checked out can arrange for that with
 the `exta_refs` field.
 - Jobs that do not want submodules to be cloned should set `skip_submodules` to `true`
+- Jobs that want to perform full clone of repo can set `clone_depth` to 0. For shallow cloning, `clone_depth` can be set to desired clone depth.
```

Comment:
I would prefer instructing people to skip setting the option at all instead of explicitly setting to 0

Reply:
thanks. Will do.